### PR TITLE
Make use of bm changes

### PR DESCRIPTION
**Pull request by odcinek**
Not yet tested

### CODE REVIEW

File: bin/grocery-delivery [hunk 1]
```diff
@@ -220,14 +220,17 @@ def setup_config
 end
 
 def get_repo
-  repo_path = File.join(GroceryDelivery::Config.master_path,
-                        GroceryDelivery::Config.reponame)
-  r = BetweenMeals::Repo.get(GroceryDelivery::Config.vcs_type, repo_path,
-                             GroceryDelivery::Log)
-  if GroceryDelivery::Config.vcs_path
-    r.bin = GroceryDelivery::Config.vcs_path
```
Comment (odcinek) [inline bin/grocery-delivery]: Is this needed? Where is it used?

File: bin/grocery-delivery [hunk 2]
```diff
@@ -254,6 +254,9 @@ else
   unless GroceryDelivery::Config.dry_run
     repo.checkout(GroceryDelivery::Config.repo_url)
```
Comment (jaymzh) [inline bin/grocery-delivery]: really? How can you call checkout on am ethod you didn't get?

File: bin/grocery-delivery (hunk 1 above)
Comment (jaymzh) [inline bin/grocery-delivery]: Yes - this is how people can override the path to git if they install it some place interesting... we used to use it when git was in /usr/local/fb-git/whateverthefuck.

**Comment by facebook-github-bot-8** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by jaymzh** [comment]
This can't be merged until facebook/grocery-delivery#17 is fixed.

**Pull request closed without merging**